yt/yt/server/lib/shell/shell_manager.cpp

```diff
@@ -400,17 +400,176 @@ IShellManagerPtr CreatePortoShellManager(
         std::move(rootInstance));
 }
 
+class TShellManager
+    : public TShellManagerBase
+{
+public:
+    TShellManager(const TShellManagerConfig& config)
+        : TShellManagerBase(config, nullptr)
+    {
+        Environment_.emplace_back(Format("HOME=%v", WorkingDir_));
+        Environment_.emplace_back(Format("G_HOME=%v", WorkingDir_));
+        auto tmpDirPath = NFS::CombinePaths(WorkingDir_, "tmp");
+        Environment_.emplace_back(Format("TMPDIR=%v", tmpDirPath));
+
+    }
+
+    NApi::TPollJobShellResponse PollJobShell(
```

Comment:
Using `NApi::` is unnecessary, since there is `using namespace NApi` in the beginning of the file.

And the similar situation with `NJobProberClient::` and `NYson::` below